### PR TITLE
Biological sampling reorder, more help text

### PR DESCRIPTION
**Pull request by jkreft-usgs**

### CODE REVIEW

File: portal_ui/static/js/portalHelp.js [hunk 1]
```diff
@@ -169,23 +169,11 @@ PORTAL.MODELS.help = {
...
-    	content : '\
-    		<div>The Assemblage is specific to biological collection data, and refers to "An association \
-    		of interacting populations of organisms in a given waterbody."  For example, macroinvertabrates or fish/necton  </div>'
-    },
     project : {
     	placement : 'auto',
     	title : 'Project Help',
     	content : '\
-    		The Project under which the sample was collected.'
+    		The Project Identifier is a designator used to uniquely identify a data collection project within a context of an organization.'
```
Comment (mbucknell) [inline portal_ui/static/js/portalHelp.js]: OCD alert: I think identifier shouldn't be capitalized.

File: portal_ui/static/js/portalHelp.js [hunk 2]
```diff
@@ -211,6 +199,27 @@ PORTAL.MODELS.help = {
         content: '\
         	<div>National Water Quality Assessment Program Samples Only (NAWQA, NASQAN, and NMN)</div' 	
     },
+    biologicalparams : {
+    	placement: 'auto',
+        title: 'Biological Sampling Parameters Help',
+        content: '\
+        	<div>These parameters are related to samples of biological organisms, such as species name or the assemblage that a group of organisms belongs to</div'
```
Comment (mbucknell) [inline portal_ui/static/js/portalHelp.js]: Need to close the div tag at the end of this line.

File: portal_ui/static/js/portalHelp.js (hunk 1 above)
Comment (mbucknell) [inline portal_ui/static/js/portalHelp.js]: One more OCD thing. The help should probably be sentences that end in periods. The last statement for Assemblage is a fragment, not  a sentence.

**Comment by mbucknell** [comment]
Should at minimum fix the div tag under biological params.

**Comment by jkreft-usgs** [comment]
fixed all comments

**Pull request merged**